ExecuteJobs.java

```diff
@@ -0,0 +1,80 @@
+import java.util.*;
+import java.io.*;
+
+public class ExecuteJobs {
+  public long start(String process) throws IOException {
+    // TODO some error handling
+    Process p = Runtime.getRuntime().exec(process);
+    long pid = p.pid();
+    return pid;
+  }
+
+  public int stop(long pid) throws IOException {
+    Process p = Runtime.getRuntime().exec("ps -q " + pid + " -o state --no-headers");
```

Comment:
I think this approach (relying on the OS to do most of the job) is sound, good pick!